src/style.css

```diff
@@ -0,0 +1,5 @@
+@import url(./css/reset.css);
```

Comment:
it's common thing to keep your stylesheet files in `styles` directory

Reply:
I move this file in folder utils